testing/endtoend/evaluators/beaconapi_evaluators/beacon_api.go

```diff
@@ -505,7 +690,7 @@ func compareJSONResponseObjects(prysmResp interface{}, lighthouseResp interface{
 func pathFromParams(path string, params []string) string {
 	apiPath := path
 	for index := range params {
-		apiPath = strings.Replace(path, fmt.Sprintf("{param%d}", index+1), params[index], 1)
+		apiPath = strings.Replace(apiPath, fmt.Sprintf("{param%d}", index+1), params[index], 1)
```

Comment:
Bug fix - wrong variable was being updated. It makes a difference for endpoints with more than 1 path param.